doc/ngsi/orion/user_and_programmer_guide/appending_data.md

```diff
@@ -69,6 +69,10 @@ Please observe the component allows considering all the records to be related to
 
 ![tOrionAppend configuration](./images/t_orion_append_conf_alt.png)
 
+Please observe as weel the Contexts tab has been used in order to centralize the configuration:
```

Comment:
Fixed in https://github.com/telefonicaid/iotp-talend-connectors/pull/14/commits/95d521020da7207f15f48a690615cc2cd6ea503b